src/current/_includes/releases/v23.2/v23.2.0-rc.1.md

```diff
@@ -0,0 +1,73 @@
+## v23.2.0-rc.1
+
+Release Date: December 21, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-rc-1-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Added a SQL function `crdb_internal.fips_ready()` that can be used to verify the [FIPS](../v23.2/fips.html) readiness of the gateway node. [#116281][#116281]
+- [Physical cluster replication](../v23.2/physical-cluster-replication-overview.html) now retries for just over 3 minutes before failing. [#116404][#116404]
+
+<h3 id="v23-2-0-rc-1-sql-language-changes">SQL language changes</h3>
+
+- `CALL` statements can now be run with [`EXPLAIN`](../v23.2/explain.html). The `EXPLAIN (OPT)` variant will show the body of the procedure, while other variants will show only the procedure name and arguments. [#116273][#116273]
+- Added support for [`IMPORT INTO`](../v23.2/import-into.html) a table that has columns typed as arrays of user-defined types (like [`ENUM`](../v23.2/enum.html)). Tables that use multiple user-defined types with the same name but different schemas are still unsupported. [#116360][#116360]
+- The [`SELECT FOR UPDATE`](../v23.2/select-for-update.html) implementation used under Read Committed isolation (and under [Serializable isolation](../v23.2/demo-serializable.html) when [`optimizer_use_lock_op_for_serializable`](../v23.2/set-vars.html) is set to `true`) now locks all [column families](../v23.2/column-families.html) instead of only the first column family. [#116828][#116828]
+
+<h3 id="v23-2-0-rc-1-operational-changes">Operational changes</h3>
+
+- The [cluster settings](../v23.2/cluster-settings.html) `storage.single_delete.crash_on_invariant_violation.enabled` and `storage.single_delete.crash_on_ineffectual.enabled` are disabled and must not be enabled. [#116897][#116897]
```

Comment:
```suggestion
- The [cluster settings](../v23.2/cluster-settings.html) `storage.single_delete.crash_on_invariant_violation.enabled` and `storage.single_delete.crash_on_ineffectual.enabled` are disabled and must not be enabled. [#116897][#116897]
```
I double-checked and we never release-noted these cluster settings when we added them, assuming it was in a 23.2 build, just FYI.

Reply:
So, there was a release note for them in the original SHA, but then the additional note removes them. I checked in with Jackson re including this and removing the other note. 

What do you think? @mdlinville

Reply:
OK cool, checked and removing.